root/release_group/Header.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+// Copyright (C) 2017 MetaBrainz Foundation
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityTabs = require('../components/EntityTabs');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const ArtistCreditLink = require('../static/scripts/common/components/ArtistCreditLink');
+const {artistCreditFromArray} = require('../static/scripts/common/immutable-entities');
+
+type Props = {
+  series: EntityT;
+  page: string;
+  entity_properties: EntityPropsT;
+};
+
+const ReleaseGroupHeader = ({rg, page, entity_properties}: Props) => {
+  const artistCredit = <ArtistCreditLink artistCredit={artistCreditFromArray(rg.artistCredit)} />;
+  const header = (
+    <div className="rgheader">
+      <h1><EntityLink entity={rg} /></h1>
+      <p className="subheader">
+        <span className="prefix">~</span> {l(`Release group by {artistCredit}`, {__react: true, artistCredit: artistCredit})}
```

Comment:
Make sure to use the same strings as the original header files, so they don't have to be retranslated: `l('Release group by {artist}', ...)`.